media/virtualDocument.ts

```diff
@@ -376,95 +452,83 @@ export class VirtualDocument {
         if (!event || !event.target) return;
         const targetElement = event.target as HTMLElement;
         const modifierKeyPressed = event.metaKey || event.altKey || event.ctrlKey;
-        // If the user presses ctrl / cmd + f we focus the search box and change the dropdown
         if ((event.metaKey || event.ctrlKey) && event.key === "f") {
+            // If the user presses ctrl / cmd + f we focus the search box and change the dropdown
             this.searchHandler.searchKeybindingHandler();
-        } else if (event.keyCode >= 37 && event.keyCode <= 40) {
-            this.arrowKeyNavigate(event.keyCode, targetElement);
+        } else if ((event.keyCode >= 37 && event.keyCode <= 40 /*Arrows*/)
```

Comment:
Ideally we would switch away from keycodes here as it's been deprecated. This is also a debt that I need to fix which will make it easier to understand what's going on here as `event.keyCode === 35` doesn't convey home as well as `event.key === "Home"`